src/ui/map.js

```diff
@@ -2582,10 +2582,15 @@ function removeNode(node) {
  * `x` and `y` properties representing screen coordinates in pixels.
  *
  * @typedef {Object} Point
+ * @example
+ * var point = Point(-77, 38);
```

Comment:
I believe that needs to be `new Point(...` and then actually needs to be called as `new mapboxgl.Point(...